packages/react-native-editor/bin/test-e2e-setup.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/bash -e
 
+# =================================================================
+# Appium Drivers
+#
+# NOTE: Please update the following versions when upgrading Appium.
+# =================================================================
+UI_AUTOMATOR_2_VERSION="2.32.3"
+XCUITEST_VERSION="5.7.0"
```

Comment:
These versions are the ones I had locally installed. So far, they haven't produced incompatibilities. In the future, we might consider updating them if necessary.